chapters/Chapter 1 — Untitled.md

```diff
@@ -18,17 +18,19 @@ Leanne turned just in time to see a crate of batteries break free of its binding
 
 Small batteries spilled from the crate as it smashed against the wall. Leanne did her best to ignore them as they pummeled her. Her legs and arms strained as she climbed into the cockpit.
 
-She reached for the _Engine Power_ switch. If she turned the engines on and managed to wrestle the controls, she could right the ship and get away. But Leanne could never get away. They were always after her, and they always would be. She could kill two today, and two more would take their place tomorrow. But if she let them live, they’d be back, and so would the two others, and so would more besides.
+She reached for the _Engine Power_ switch. If she turned the engines on and managed to wrestle the controls, she could right the ship and get away. But Leanne could never get away: they were always after her, and they always would be. There were at least two after her today, and there’d be at least two more tomorrow, and two more after that. 
 
-Leanne withdrew her hands. Three years ago, she had woken up in a Faces facility, bloody scalpel in hand, dead bodies scattered around her, disoriented and without memory. Naturally, she had stolen just about everything she could carry—which was quite a bit, particularly after she had stolen a cart.
+“No reprieve,” she muttered. It was necessary. Getting away was not sufficient. Thirty seven of them had to corner her on Delphinia before she could admit it to herself: they could survive, or she could. They did not survive.
```

Comment:
The wording of this has improved since my first iteration but I can make it better still.

"Thirty seven of them had to corner her on Delphinia before she had admitted it to herself: they could survive, or she could."